File: python/wget.py
#!/usr/bin/python3

import subprocess
#pipe = subprocess.Popen("wget -O index.html http://cslab.jbnu.ac.kr/ ", shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)

file = open("index.html", "r")
read = file.read()
print(read)

File: php/wget.php
#!/usr/bin/php

<?
#$html = popen("wget -O index.html http://cslab.jbnu.ac.kr/", "r");
	
$file = fopen("./index.html", "r");
$read = fread($file, 10000);

preg_match_all('/\w+/', $read, $matches);

print_r($matches[0]);
$si = sizeof($matches[0]);
echo "$si\n";
#pclose($html);
fclose($file);
?>
